README.rst

```diff
@@ -14,18 +14,78 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.2.6 or newer.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
+
+::
+
+    $ pip install omero-figure
+
+Add figure custom app to your installed web apps:
+
+::
+
+    $ bin/omero config append omero.web.apps '"omero_figure"'
+
+Display a link to 'Figure' at the top of the webclient:
+
+::
+
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "figure"}]' 
+
+Now restart OMERO.web as normal.
+
+**Warning**:
+
+OMERO.figure version 2.x is meant to be installed into OMERO.web **5.2.6 or newer**.
+This is due to Django Framework compatibility and a package re-organization required to distribute the application using Python Package Manager PyPi.
+
+
+Enabling PDF generation
+-----------------------
 
-Please see instructions at http://figure.openmicroscopy.org
+This following steps need to perform where the OMERO.server is installed.
 
+* Install `reportlab <https://bitbucket.org/rptlab/reportlab>`_ PDF python package:
+
+::
+
+    $ pip install reportlab
+
+* Install script to generate figure. To perform this task, you must be an admin.
```

Comment:
Minor suggestion: If there's now 2 alternative options for install, it's nice to give a heads-up of this before the first option. It's annoying to follow install instructions and then find that there's an alternative *after* you've already done it one way.